infinite_tracing/test/test_helper.rb

```diff
@@ -16,6 +16,7 @@ module NewRelic; TEST = true; end unless defined? NewRelic::TEST
 require 'minitest/autorun'
 require 'minitest/pride' unless ENV['CI']
 require 'mocha/setup'
+require_relative '../../test/minitest/test_time_plugin'
```

Comment:
Ideally I'd like to have `require` and `require_relative` lines split into 2 sections and have each section be alphabetized based on gem / path string values. Is this a dependency order thing, where minitest needs to be ready and then we need to do something before loading the agent itself?

Reply:
Resolved by: 56b03baae0646ad361243f98d8ed9594277cc285